implementation/choreography-to-orchestration/README.md

```diff
@@ -0,0 +1,131 @@
+# Convert Choreography to Orchestration
+This project is the CDK implementation of ['Convert Orchestration to Choreography'](https://serverlessland.com/refactoring-serverless/choreography-to-orchestration) pattern. It shows how you can convert a distributed message flow Choreography architecture to a central workflow Orchestration architecture. 
+
+## How it works: 
+The use case is for an application that place an order for a product (Order Placement Workflow). Customer sends a request to place an order for a product. The application completes a series of steps reuired to complete an order and ship the product to customer. 
+
+***
+## Choreography: 
+
+The flow starts with client submitting a place order request to the API Gateway REST API. Placing order is a lengthy process so there is no immediate response. The API accepts the place order request and returns a success or failed response. 
+
+API gateway then sends the request to a lambda function, which processes the payment and persists the payment data in an Amazon DynamoDB table. After that, the request is published to a 'Ship Order' SNS topic. Using Publish-subscribe model another lambda function will register to the 'Ship Order' SNS topic. This function reads the data from DynamoDB table to verify if payment was processed successfully and then ships an order. Throws an error is payment is not processed successfully. 
```

Comment:
lambda needs to have capital "L" throughout this doc

Reply:
Updated